presto-docs/README.md

```diff
@@ -0,0 +1,103 @@
+# presto-docs - Presto Documentation
+
+The presto-docs module contains the reference documentation for Presto.
+
+## Tools
+
+The default build of the docs is performed with Apache Maven.
+
+Documentation source files can be found in [Restructured
+Text](https://en.wikipedia.org/wiki/ReStructuredText) (`.rst`) format in
+`src/main/sphinx` and sub-folders.
+
+The engine used to create the documentation in HTML format is the Python-based
+[Sphinx](https://www.sphinx-doc.org).
+
+## Default Build
+
+The default build is using Apache Maven and Java like for the rest of the
+Presto build. You just need to have built the current version from the root.
+Subsequently you can build the site:
+
+```bash
+mvn -pl presto-docs clean install
```

Comment:
we recommend `./mvnw` (maven wrapper which ensures that everyone runs the same maven version)
the `./mvnw` file is part of the repo

```suggestion
./mvnw -pl presto-docs clean install
```

i understand this becomes awkward (`../mvnw` -- does it work?) in the option with `cd presto-docs` ... maybe simplify and remove the second option? or, do you find it more convenient that way?

Reply:
Tested and updated. I find the working in the local folder quite a bit easier to be honest

Reply:
Also added note about alternatively using mvn